releases/v1.1-alpha.20170803.md

```diff
@@ -57,7 +57,7 @@ Get future release notes emailed to you:
 
 ### Bug Fixes
 
-- The first statement (or batch of statements) in a transaction can no longer return "retry transaction" errors; the server will retry the statement automatically. [#16874](https://github.com/cockroachdb/cockroach/pull/16874)
+- Retryable errors that were encountered by the first batch of statements in a transaction but were deferred and only returned to the client at COMMIT time are now handled transparently by no longer deferring them and automatically retrying the first batch. [#16874](https://github.com/cockroachdb/cockroach/pull/16874)
```

Comment:
I think this is hard to follow because it introduces a lot of new concepts (deferred errors?). How about something like "Transactions that encounter a retryable error in their first batch of statements now retry automatically"? I don't think we need to be perfectly precise here; the PR link is there if anyone needs more details.